bin/consul-manage

```diff
@@ -8,6 +8,9 @@ set -eo pipefail
 preStart() {
     _log "Updating consul advertise address"
     sed -i "s/CONTAINERPILOT_CONSUL_IP/${CONTAINERPILOT_CONSUL_IP}/" /etc/consul/consul.hcl
+
+    _log "Updating consul datacenter name"
+    sed -i "s/CONSUL_DATACENTER_NAME/${CONSUL_DATACENTER_NAME:-dc1}/" /etc/consul/consul.hcl
```

Comment:
The environment variable make it straightforward to manually set the datacenter and makes testing and iteration easier, but it doesn't incorporate the automatic datacenter detection that seems to be described in #23.

I can address that part of #23 if we can decide on how it would be triggered compared to the method implemented in this PR for manual DC specification.

Options seem to include:
 - `CONSUL_DATACENTER_NAME=*AGREED UPON STATIC VALUE*` is checked specifically in `preStart` and triggers a metadata check if it matches. Anything but the static value would be assumed to be a manually-specified DC
- `CONSUL_DATACENTER_AUTODETECT=triton` would invoke triton-specific attempts to discover the current datacenter and allow for future implementations for other cloud providers
- `CONSUL_DATACENTER_AUTODETECT=1` would attempt to check different methods of discovering the current datacenter, though involve more "magic" (i.e. brittle or potentially-unexpected behavior)
- `CONSUL_DATACENTER_AUTODETECT_TRITON=1` would invoke triton-specific attempts to discover the current datacenter, establishing the pattern for other providers to user their own envs
- something else?

Reply:
I agree, and the env var satisfies non-Triton use cases. I wasn't entirely set on using metadata but we could marry the two ideas for convenience.

- Use `CONSUL_DATACENTER_NAME` to manually set/force the datacenter name.
- If env var isn't present AND we have `/native/usr/sbin/mdata-get` then poll metadata for `sdc:datacenter_name`.
- Finally, use the default `dc1` if none of the above conditions gave us a custom value.

Reply:
Falling back to automatic detection seems like the best option for ease-of-use, great suggestion!

I'll include the steps taken in the `README` so users can see how it'll work without having to dig into the source.